docs/local-site.yml

```diff
@@ -0,0 +1,17 @@
+site:
+  title: AeroGear Docs Site
+  start_page: android-sdk::index
+content:
+  sources:
+  - url: ../
+    branches: AEROGEAR-2512
```

Comment:
Is this the correct value for branch?  I feel like it should be master.

Reply:
@secondsun chances are,  this will never be the 'correct value', it's only for a local preview. The normal workflow I see is to create a branch for your work, then change this value to your branch name and validate your work as you proceed. This will all be documented in https://github.com/aerogear/processes/tree/master/modules/docs/pages 
... making it up as I go along ;)

Reply:
Was looking at the antora docs and it they say if you use HEAD then you will always pick the current branch for the project.  Is this a solution?

Reply:
@secondsun  that's perfect solution, thanks 
(glad someone reads docs)